upgrade/helpers/tasks.py

```diff
@@ -663,9 +663,9 @@ def capsule_sync(cap_host):
     job_execution_time("Capsule content sync operation", start_time)
 
 
-def katello_restart():
-    """Restarts the katello services"""
-    services = run('katello-service restart')
+def foreman_service_restart():
+    """Restarts the foreman-maintain services"""
```

Comment:
@devendra104 this is misleading, you can change description to something like:
```suggestion
    """Restarts the foreman/satellite services using foreman-maintain"""
```

Reply:
@jameerpathan111 why it is misleading? here we are stopping all the service via foreman-maintain, that's the reason we mentioned foreman-maintain.

Reply:
No problem, I just wanted it to sound like we are restarting foreman/Satellite services.